src/schema/v2/__tests__/viewingRoomConnection.test.js

```diff
@@ -0,0 +1,132 @@
+import { runQuery } from "schema/v2/test/utils"
+import gql from "lib/gql"
+
+// TODO: skip until USE_UNSTITCHED_VIEWING_ROOM_SCHEMA is set to true and the schema is updated
+describe.skip("ViewingRoomConnection", () => {
```

Comment:
You can stub config and set it to true just for purpose of this test (same for above test)

Reply:
thanks, done!